docs/source/interface.rst

```diff
@@ -20,6 +20,7 @@ The following is a complete list of methods on PyFilesystem objects.
 * :meth:`~fs.base.FS.getdetails` Get details info namespace for a resource.
 * :meth:`~fs.base.FS.getinfo` Get info regarding a file or directory.
 * :meth:`~fs.base.FS.getmeta` Get meta information for a resource.
+* :meth:`~fs.base.FS.getmodified` Get info regarding the last modified time of a resource.
```

Comment:
Should this have been "Get the last modified time of a resource."? The "info regarding" seems superfluous?

Reply:
@lurch : Good catch, I think I'm due for a documentation PR next.